interfaces/admin_cli.py

```diff
@@ -0,0 +1,511 @@
+        else:
+            return "invalid_input"
+
+    def preloop(self) -> None:
+        """
+        Display plan menu.
+        """
+        self.plan_menu()
+
+    def return_previous_page(self):
+        while True:
+            option = input("Enter R to return back to the previous menu.\n > ")
+            if option.upper() == 'R':
+                PlanMenu(self.user).cmdloop()
+            else:
+                print("\033[31m {}\033[00m".format("** Invalid input. Enter R to return back to main menu."))
+                continue
+
+    # ----- basic commands -----
+    def do_create_plan(self, arg):
+        """
+        #1 Create a plan
+        """
+        print("\033[100m\033[4m\033[1m{}\033[0m ".format("Create a new emergency plan"))
+        e_name = input("Enter emergency plan name: ")
+        # -- handle emergency type
+        print("Enter a emergency type from the list")
+        emergency_types = plan_controller.list_emergency_types()
+        e_types_dict = {t.value: t for t in emergency_types}  # create a dictionary for Enum
+        for key in e_types_dict.keys():
+            print(f'- {key}')
+        while True:
+            e_type = input(">")
+            try:
+                emergency = e_types_dict[e_type]
+                e_type: Plan.EmergencyType = emergency
+                break
+            except KeyError:
+                print(f'Type {e_type} is not on the list. Please re-enter type:')
+
+        e_description = input("Enter description: ")
+        e_geo_area = input("Enter geographical area: ")
+        while True:
+            camps_input = input("Enter camp names (use comma to separate camps): ")
+            camp_names = camps_input.split(",")
+            if camp_names == '':
+                print("\033[31m {}\033[00m".format("** Camp name cannot be empty."))
+                continue
+            try:
+                camps = [plan_controller.create_camps(name) for name in camp_names]
+                break
+            except ControllerError:
+                print("\033[31m {}\033[00m".format("** Failed to create camps. Please check and re-enter"))
+                continue
+        try:
+            plan_controller.create_plan(plan_name=e_name,
+                                        emergency_type=e_type,
+                                        description=e_description,
+                                        geographical_area=e_geo_area,
+                                        camps=camps)
+            print("\x1b[6;30;42m success! \x1b[0m\t")
+            print(f"Plan {e_name} created.")
+            self.return_previous_page()
+        except ControllerError:
+            print("\033[31m {}\033[00m".format("** Unable to create an emergency plan. Press R to return and retry."))
+            self.return_previous_page()
+
+    def do_list_plans(self, arg):
+        """
+        #2 List out all the existing plans
+        """
+        plan_controller.list_plans()
```

Comment:
I was just testing out the Admin menu and noticed that when you try to list all emergency plans, nothing is returned. 
When I was making the list_plans function in plan_controller, one of the notes was that "this would not be shown in the menu" (from the initial set up of the controllers) so I believe that this function is not supposed to be used here. 
If, of course, we wanted to display all the plan names as a menu option, then we could modify the list_plans function (or create a new function in plan_controller) to do this. If that's something that you require, then I could work on this!

Reply:
I think a `print` is missing here
```python
print(plan_controller.list_plans())
```

Reply:
I believe if we use print then it will actually give an output like this: 
[<models.plan.Plan object at 0x0000015158A9A0A0>, <models.plan.Plan object at 0x0000015158A9A340>, <models.plan.Plan object at 0x0000015158A9AFD0>]
(the above is from a trial I did in the test_plan_controller file a while back) 
So it will not print the plan names.